README.md

```diff
@@ -62,6 +65,66 @@ dsc_windowsfeature {'IIS':
 
 All DSC Resource names and parameters have to be in lowercase, e.g: `dsc_windowsfeature` or `dsc_name`.
 
+### Installing Packages with DSC
+
+You can install MSIs with DSC using the Puppet type `dsc_package` which maps to the `Package` DSC Resource.
+
+~~~puppet
+dsc_package{'installpython'
+  dsc_ensure    => 'Present',
+  dsc_name      => 'Python 2.7.10',
+  dsc_productid => 'E2B51919-207A-43EB-AE78-733F9C6797C2'
+  dsc_path      => 'C:\\python.msi',
+}
+~~~
+
+The `Package` DSC Resource requires the following pieces of information to install an MSI:
+
+- ProductName: The `Name` of product being installed.
+- ProductId: The `ProductCode` of the MSI, which is the unique identifier for that MSI version.
```

Comment:
Side not, I'm surprised DSC uses `ProductCode`, because you can't ensure a particular hotfix has been installed. To do that you need to ensure a particular `PackageCode` is installed, which is something that Puppet can do: https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/package/windows/msi_package.rb#L48-L49, e.g.